makeflow/src/dag.c

```diff
@@ -420,5 +421,17 @@ int dag_local_jobs_running( struct dag *d )
 	return itable_size(d->local_job_table);
 }
 
-
+uint64_t dag_absolute_filesize( struct dag *d )
```

Comment:
This function counts files even if they have been garbage collected. Is that ok?